PowerEditor/src/NppDarkMode.h

```diff
@@ -48,6 +48,7 @@ namespace NppDarkMode
 	{
 		bool enable = false;
 		bool enableMenubar = false;
+		bool enablePlugin = false;
```

Comment:
I would say, let's enable this option by default. 
And remove the option from  GUI. 
If users complain, we could add back the option on GUI, or a non-GUI option in config.xml.

Reply:
Ok.